### ISSUE TITLE
Where did the ember-data API docs go?

### ISSUE THREAD

**Issue by kbullaughey** [emberjs/website#2955]
Not sure if this is the right place to ask, but I can't find the docs for the DS namespace on https://emberjs.com/api anymore. The layout recently changed, and there doesn't seem to be any ember-data documentation anymore. Perhaps I'm being dense, but I can't find it, and if it's still there, might a link to it be made more obvious on the API documentation page?

**Comment by qsvui** [comment]
Seems like the docs have been broken. For example, the entry for Ember.Object is now empty. The search input that was in the upper right hand corner of the site is also gone.

**Comment by locks** [comment]
Hi, we just launched the new [ember-api-docs](https://github.com/ember-learn/ember-api-docs/) live at EmberCamp and I explained some of the work that will happen in the next few days.

The "Projects" section in the top-left of the API docs was missing due to a bug.
The search is also a known limitation, because we to index through Algolia to get the search results. There is a sort of chicken and egg problem though, as we need to publish the website for Algolia to be able to index it, so we're taking care of that and should bring search back within the week.
